receiver/kafkareceiver/json_unmarshaler_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkareceiver
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewJSONUnmarshaler(t *testing.T) {
+	t.Parallel()
+	um := newJSONLogsUnmarshaler()
+	assert.Equal(t, "json", um.Encoding())
+}
```

Comment:
Do you mind adding some tests to validate that a given json blob of `{blah ...}` is encoding as a plog value?